### ISSUE TITLE
Zoom screen 1.4.4 Resize Text Accessibility issue

### ISSUE THREAD

**Issue by ghahemcl** [ponlawat-w/moodle-local_accessibility#53]
I have noticed an accessibility issue. At approximately 300 percent zoom, the site‑wide accessibility widget extends beyond the viewport and cannot be fully accessed or operated. This results in loss of functionality for users who rely on high zoom, including low‑vision users. 

<img width="1904" height="938" alt="Image" src="https://github.com/user-attachments/assets/34816fda-ee32-49ec-9565-f6ecc5f7af6e" />

Browser tested:

Chrome Version 147.0.7727.116 (Official Build) (64-bit)
Version 147.0.7727.116 (Official Build) (64-bit)

Please would you be able to look into this as a fix for a future plugin update.
